doc: include followTags and GPG=TTY mention (#52906)
nodejs/node: doc/contributing/releases.md

npm install -g git-secure-tag
```

> Ensure to disable `--follow-tags` in your git settings using: `git config push.followTags false`

> If your GPG key is protected by a password, you might need to run:

Reviewer: ```suggestion
If your private key is protected by a passphrase, you might need to run:
```

> ```console
> $ export GPG_TTY=$(tty)

Reviewer: You can simplify copy/pasting by removing the `$`
````suggestion
```bash
export GPG_TTY=$(tty)
````

```

before creating the tag.

> To create a tag using the following command:

Reviewer 2: ```suggestion
To create a tag use the following command:
```
